django_slack_oauth/views.py

```diff
@@ -57,15 +59,22 @@ def get(self, request, *args, **kwargs):
         if not api_data['ok']:
             return self.error_message(api_data['error'])
 
+        pipelines = settings.SLACK_PIPELINES
```

Comment:
It would be better to call this as a separate helper function or method like `apply_pipelines` or smth.